src/netbase.cpp

```diff
@@ -694,19 +696,19 @@ CNetAddr::CNetAddr(const struct in6_addr& ipv6Addr)
     SetRaw(NET_IPV6, (const uint8_t*)&ipv6Addr);
 }
 
-CNetAddr::CNetAddr(const char* pszIp, bool fAllowLookup)
+CNetAddr::CNetAddr(const char *pszIp)
```

Comment:
nit: I think that the convention of placing asterisks right next to the variable type for pointers (i.e. `const char* pszIp`) is preferred almost everywhere in the codebase (e.g. in this very same diff, in netbase.h). We should strive for consistency.

Reply:
yeah, these changes are a result of the cherry-pick from upstream. I can go back through and edit each commit to NOT make this change.

This, of course, opens up a debate topic on what we want to use as "standard" and include in the developer notes.

Reply:
Yeah, in C++ this is probably a debate as old as "tab vs spaces".
I think we should go with the most used convention in the codebase (without necessarily specify it in the contribution guidelines) and, in any case, at least keep consistency inside a single PR (e.g. it doesn't make sense to keep a function signature as `CNetAddr(const char* pszIp)` in the header, but change it in the *.cpp to make it `CNetAddr(const char *pszIp)`).

Reply:
updated to remove the reference side change